_pydev_bundle/pydev_console_utils.py

```diff
@@ -90,10 +82,6 @@ def readline(self, *args, **kwargs):
     def write(self, *args, **kwargs):
         pass  # not available StdIn (but it can be expected to be in the stream interface)
 
-    def isatty(self):
-        # return True for IPython console only
-        return USE_IPYTHON
```

Comment:
Patch seems good, just one minor comment: instead of removing 'def isatty' altoghether, it'd be nice to let it there with the code:

```
def isatty(self):
    return self.original_stdin.isatty()
```

to avoid in letting it go to __getattr__ (ideally, we should implement all that's defaut to stdin and just let odd things go to __getattr__, which is generally slower and also hides a bit what we're actually wrapping).